packages/api/AlvTime.Business/EconomyData/EmployeeSalary.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace AlvTime.Business.EconomyData
+{
+    public record EmployeeSalary(int UserId, decimal HourlySalary, DateTime FromDate, DateTime? ToDate, int Id) : 
+        EmployeeSalaryRequest(UserId, HourlySalary, FromDate, ToDate)
```

Comment:
Trenger EmpoyeeSalary å arve fra EmployeeSalaryRequest eller blir det bare mer forvirrende?

Reply:
Nei, egentlig burde det jo være omvendt, og så er jeg litt lite glad i å ikke koble sammen objekter som åpenbart har noe med hverandre å gjøre. Fjernet arv, for jeg er enig :)